wasm/__init__.py

```diff
@@ -2879,9 +2880,10 @@ def spec_growtable(tableinst, n):
     if verbose >= 1:
         print("spec_growtable(", ")")
     len_ = n + len(tableinst["elem"])
-    if len_ > 2 ** 32:
+
+    if len_ > constants.UINT32_CEIL:
```

Comment:
Is this off by one? if `len_` needs to fit into a 32 bit uint, then it seems it should be `>=` here.